fs-ansible/roles/lustre/tasks/format.yaml

```diff
@@ -40,8 +40,15 @@
         state: present
       loop: "{{ mdts.keys() }}"
 
+    - name: Wait for MDT partition to appear in /dev
+      wait_for:
+        path: "/dev/{{ item.key }}p1"
+        timeout: 30
+        sleep: 5
+      loop: "{{ mdts|dict2items }}"
+
     - name: Reformat MDTs
-      command: "/usr/sbin/mkfs.lustre --mdt --reformat --fsname={{ fs_name }} --index={{ item.value }} --mgsnode={{ mgsnode }}{{ lnet_suffix }} /dev/{{ item.key }}p1"
+      shell: "test -b /dev/{{ item.key }}p1 && /usr/sbin/mkfs.lustre --mdt --reformat --fsname={{ fs_name }} --index={{ item.value }} --mgsnode={{ mgsnode }}{{ lnet_suffix }} /dev/{{ item.key }}p1"
```

Comment:
Do we really need this bit?

Reply:
Not really, it seemed like a good idea before the "wait" checks. Gone.